src/docs/frontend/development-server.mdx

```diff
@@ -27,15 +27,9 @@ If you see an error similar to the below when accessing the development server:
 
 ![your connection is not private example](../img/connectionNotPrivate.png)
 
-You can either grant a temporary exception in your browser, or create and install a local certificate and use your OS to mark them as "trusted". The instructions below will show you how to create and install a local certificate.
+You can either grant a temporary exception in your browser, or create and install a local certificate and use your OS to mark them as "trusted". SSL is required for the frontend development service, since the production API is served over https.
```

Comment:
```suggestion
You can either grant a temporary exception in your browser, or create and install a local certificate and use your OS to mark them as "trusted". TLS is required for the frontend development service, since the production API is served over https.
```